operations/helm/charts/alloy/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ internal API changes are not present.
 Unreleased
 ----------
 
+### Enhancements
+
+- Update jimmidyson/configmap-reload to 0.14.0. (@duncan485)
```

Comment:
```suggestion
- Update jimmidyson/configmap-reload to 0.14.0. (@petewall)
```

Reply:
The most unfortunate copy and paste error! Thanks!